Formula/open-zwave.rb

```diff
@@ -1,132 +1,32 @@
 class OpenZwave < Formula
   desc "Library for selected Z-Wave PC controllers"
-  homepage "http://www.openzwave.com"
-  url "http://old.openzwave.com/downloads/openzwave-1.2.919.tar.gz"
-  sha256 "473229f3dd3d6b260e6584b17e5c5f2e09e61805f89763f486a9f7aa2b4181ba"
+  homepage "http://www.openzwave.net/"
+  url "https://github.com/OpenZWave/open-zwave/archive/v1.4.tar.gz"
+  sha256 "efb54adf0cda12f76fcb478f7cf3b7d108b5daf38d60b4410cca7600aa072e83"
+  head "https://github.com/OpenZWave/open-zwave.git"
 
-  bottle do
-    sha256 "ee97a2cce9fed3f63fb917e09e448f35815eff5120ebae16711777be4796d5c0" => :high_sierra
-    sha256 "68502b64e9fb7031f9fc5fe4fa7e6714fd5e5ab93d467169e7e47e90c322ff64" => :sierra
-    sha256 "4bf1d8a8ba3fcf4ee39df9bfe09017d3a432047e5e6bf0dce6f6e612ad174b95" => :el_capitan
-    sha256 "9ceb267d8fb564daefb535f65ac71f426535f7552b9feffcb3638793f0c40810" => :yosemite
-    sha256 "753eb6cb76dd0c170c1e84285a702fb9fb49c76aa8d59970ecea2160938f0bba" => :mavericks
-    sha256 "be2b4c51703ee93865a6c755a4394b6271eb599ea98a7431b54d090276a0e9f7" => :mountain_lion
-  end
-
-  # Patch to build a .dylib instead of a .so
-  # This patch has been merged upstream and can be removed after the next release:
-  # https://code.google.com/p/open-zwave/source/detail?r=954
-  patch :DATA
+  depends_on "pkg-config" => :build
+  depends_on "doxygen" => :build
 
   def install
     ENV["BUILD"] = "release"
     ENV["PREFIX"] = prefix
-
+    ENV["pkgconfigdir"] = "#{lib}/pkgconfig"
```

Comment:
This shouldn't be necessary

Reply:
I wish it wasn't but it is pretty messy.
https://github.com/OpenZWave/open-zwave/blob/master/cpp/build/support.mk#L103-L108

The only other way is to add `mkdir "#{lib}/pkgconfig"` before the build which looks worse to me

Reply:
This needs to be reported and fixed by upstream.